src/api/model/MappingModel.ts

```diff
@@ -0,0 +1,25 @@
+/* Import Types */
+import { Mapping as MappingType, Dict } from "global/Types"
+
+
+/* Source System Model for API calls */
+const MappingModel = (result: Dict) => {
+    const mapping: MappingType = {
+        id: result.id,
+        version: result.attributes.version,
+        created: result.attributes.created,
+        creator: result.attributes.creator,
+        mapping: {
+            name: result.attributes.mapping.name,
+            description: result.attributes.mapping.description,
+            mapping: {
```

Comment:
Did you ask Sou if she could change it. Feel free to create a user story for it. I would say -> `mapping` -> `field_mapping`

Reply:
Yes, I proposed it to Sou and she was ok with it, will add it to Jira so we do not forget
https://naturalis.atlassian.net/browse/DD-371?atlOrigin=eyJpIjoiYTIwMGE0M2I0NmM3NDllMTkyZjM3ZDEwNmQwZGM1YjEiLCJwIjoiaiJ9